packages/core/src/types/IKeyManager.ts

```diff
@@ -81,6 +81,35 @@ export interface IKeyManagerDecryptJWEArgs {
   data: string
 }
 
+/**
+ * Input arguments for {@link IKeyManager.keyManagerSign | keyManagerSign}
+ * @public
+ */
+export interface IKeyManagerSignArgs {
+  /**
+   * Key ID
+   */
+  kid: string
```

Comment:
i have probably looked at JOSE specs for too long, that is why i automatically think this is the value that goes into a JWE or JWS or JWK. if we used something for internal reference only but is different from what we'd put into the JOSE objects, then i would feel more comfortable if we just said `keyHandle` or `keyReference`.

Reply:
good point.
My thinking here was to align to the `IKey` interface, but since that one has the same problem you're signaling, I can be more progressive and name this param better in advance

Reply:
done. renamed to `keyRef`